.circleci/config.yml

```diff
@@ -1,91 +1,49 @@
-# Ruby CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-ruby/ for more details
-#
-version: 2
+version: 2.1
+orbs:
+  gem: zfhui/ruby-gem@0.1.0
```

Comment:
Orb zfhui/ruby-gem@0.2.1 is out, with which you can also release a gem to RubyGems (workin on the Gemfury part :) ). Feel free to give it a go, if you want, but I'm also fine with this cleaning up.